docs/reference/6-data-import/technical-articles/hotfixes/di-1270.md

```diff
@@ -0,0 +1,43 @@
+# DI-1270
+
+# Overview
+
+This page provides information for the Data Import DI-1270 hotfix.  During the upgrade between Data Import 1.4.0 and Data Import 2.0.0 for multi-threading and performance updates, logging strategies where updated away from log4net to [Serilog](https://serilog.net/) and [Microsoft.Extensions.Logging](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging?view=dotnet-plat-ext-7.0) for architectural and performance reasons.  In Data Import 2.0.0, the Ingestion logging is using Microsoft.Extensions.Logging and the log messages are queued at the end of each job run to help accommodate multi-threading.  In field usage, it's been found that ingestion logging is consuming the majority of system RAM for large data loads, which has impacted the dependability of these data loads.
+
+This hotfix exposes a configuration setting for Ingestion logging levels, which will allow the implementer filter messages to the database as desired amongst existing levels (Debug, Information, Notice, Warning, Error, Critical), in attempt to reduce down the number of build-up of log messages during large data loads.  The field experiences from this hotfix will be considered into future releases into Data Import based on results.
+
+# Related Tickets
+
+ [EDFI-2159](https://tracker.ed-fi.org/browse/EDFI-2159?src=confmacro)
+
+ [DI-1270](https://tracker.ed-fi.org/browse/DI-1270?src=confmacro)
+
+ [DI-1275](https://tracker.ed-fi.org/browse/DI-1275?src=confmacro)
```

Comment:
Now that the public does not have access to these issues, we should remove the links to them.